source/release-notes/4-0-0.rst

```diff
@@ -64,7 +70,19 @@ New Features
 Updates
 ^^^^^^^
 
-* Upgraded Elasticsearch version to 7.16.2
+* Unlock permission added and can be assigned to a role |br|
+  See `here <https://github.com/craftercms/craftercms/issues/3690>`__ for more information
+
+* Added support for deployment mode for deployer processors |br|
+  Supported modes currently are ``PUBLISH`` and ``SEARCH_INDEX``.
+  See :ref:`here <crafter-deployer-api-target-deploy>` and :ref:`here <crafter-deployer-api-target-deploy>`
+  for more information
+
+* Upgraded Tika to 2.3.0
+
+* Upgraded SpringBoot to 2.6.4
+
+* Upgraded Elasticsearch version to 7.16.3
```

Comment:
Upgraded Elasticsearch to version 7

Reply:
Remove the "user warning when deleting" comment
Remove the JS library comment and change it to indicate the SDK is updated.

Add the XB SDK to the new features added and check with Roy on the package name and link.